.circleci/config.yml

```diff
@@ -7,6 +7,7 @@ jobs:
       - image: silex/emacs:25.1
```

Comment:
Try changing this to silex/emacs:master-dev

Their dev variants come with cask and python installed already. That means we can probably also get rid of the cask installation line below.